backend/src/main/java/com/google/sps/datastore/ListStringConversion.java

```diff
@@ -14,28 +14,18 @@
 
 package com.google.sps.datastore;
 
-import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
-import java.util.StringTokenizer;
 
 public class ListStringConversion {
```

Comment:
Now that these methods are shorter, WDYT about making them private static methods in the datastore class? 

It's ok if the test has to duplicate the logic to parse the String stored in the datastoreservice. The test is allowed to make some assumptions about implementation details of the unit under test.